subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -269,7 +276,7 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
         given:
         buildFile << """
             plugins {
-                id "org.xtext.xtend" version "1.0.5"
+                id "org.xtext.xtend" version "1.0.17"
```

Comment:
Their documentation might just be outdated. I'd upgrade to the latest version. @oehme might know more as he was involved in the implementation of the plugin I believe.